### ISSUE TITLE
Multiple text widgets in the same line

### ISSUE THREAD

**Issue by dustContributor** [vurtun/nuklear#536]
Hi! [My last issue](https://github.com/vurtun/nuklear/issues/487) didn't gather any responses but I'm guessing that want I want to do can be resolved in a different way.

Rather than having a widget with dynamic columns, I just want to place multiple text widgets in the same line. How do I do that? Having in mind that the text widgets could be each of different widths, and that each line could have any number of text widgets.

This is so I can do text like:

"This is some normal text with **colored text** in between that could
 have **many lines of text** in the middle and **_other kinds too_** "

In that example I'd have:
```
row1
  normal_text("This is some normal text with ")
  colored_text("colored text")
  normal_text(" in between that could")
row2
  normal_text("have ")
  colored_text("many lines of text")
  normal_text(" in the middle and ")
  cursive_text("other kinds too")
```
Makes sense?

**Comment by newobj** [comment]
Did you ever figure this out? I want to do the exact same thing.

**Comment by dustContributor** [comment]
Eh sort of, I ended up pre-computing how many text widgets I'll have in a single line and do a loop, also I had to calculate how wide each text widget would be using the font data (with stb_truetype library), to set each column width. More or less I had to recreate all of nk_text_wrap on my side.
```
for each line in textLines
  nk_layout_row_begin( ctx, Nuklear.NK_STATIC, height, line.segmentCount )
  for each textSegment in line.textSegments
    nk_layout_row_push( ctx, textSegment.width )
    nk_text(ctx, textSegment.content)
  nk_layout_row_end(ctx);
```
It's not perfect because the column padding is a bit wider than the spacing between text widgets, so if I have a line that is "Some normal text and **some colored text**", the colored part will be slightly farther than it should from the normal part.
